py-rse/style.Rmd

```diff
@@ -11,9 +11,34 @@ This is particularly true when we look at programming style.
 Computers don't need to understand programs in order to execute them,
 but people do if they are to create, debug, and extend them.
 
-In the preceding chapters we wrote code to analyze word counts in classic novels.
+In the preceding chapters we wrote code to analyze word counts in classic novels,
```

Comment:
no need for comma at end of line. (holy nitpicking, greg)

Reply:
the more nitpicking now, the less later...